.github/workflows/verify_install_arm64.yaml

```
@@ -0,0 +1,88 @@
+# © 2023 SolarWinds Worldwide, LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at:http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+name: Verify Installation ARM64
+
+on:
+  workflow_dispatch:
+    inputs:
+      install-registry:
+        required: true
+        description: 'Registry used for install tests, e.g. pypi, testpypi, packagecloud'
+        type: choice
+        default: 'pypi'
+        options:
+        - pypi
+        - packagecloud
+        - testpypi
+      solarwinds-version:
+        required: false
+        description: 'Optional solarwinds-apm version, e.g. 0.0.3.2'
+
+env:
+  SOLARWINDS_APM_VERSION: ${{ github.event.inputs.solarwinds-version }}
+  SW_APM_COLLECTOR_AO_PROD: ${{ secrets.SW_APM_COLLECTOR_AO_PROD }}
+  SW_APM_COLLECTOR_PROD: ${{ secrets.SW_APM_COLLECTOR_PROD }}
+  SW_APM_COLLECTOR_STAGING: ${{ secrets.SW_APM_COLLECTOR_STAGING }}
+  SW_APM_SERVICE_KEY_AO_PROD: ${{ secrets.SW_APM_SERVICE_KEY_AO_PROD }}
+  SW_APM_SERVICE_KEY_PROD: ${{ secrets.SW_APM_SERVICE_KEY_PROD }}
+  SW_APM_SERVICE_KEY_STAGING: ${{ secrets.SW_APM_SERVICE_KEY_STAGING }}
+
+jobs:
+  launch-arm64:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.launch.outputs.matrix }}
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.CI_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.CI_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - id: launch
+        uses: solarwindscloud/ec2-runner-action@main
+        with:
+          action: launch
+          matrix: |
+            py3.7-ubuntu18.04
+            py3.11-ubuntu20.04
+          github-token: ${{ secrets.CI_GITHUB_TOKEN }}
+          runner-user: github
+          runner-directory: /github/actions
+          instance-type: t4g.medium
+          ami-name: gha-arm64-ubuntu22-.*
+          ami-owner: "858939916050"
+          subnet-id: subnet-0fd499f8a50e41807
+          security-group-ids: sg-0fd8d8cd6effda4a5
+
+  install-tests:
+    needs:
+      - launch-arm64
+    strategy:
+      matrix:
+        job:
+          - py3.7-ubuntu18.04
+          - py3.11-ubuntu20.04
+    runs-on: ${{ fromJSON(needs.launch-runners.outputs.matrix)[matrix.job].label }}
+    steps:
+      - run: echo "Hello from EC2 ${{ matrix.job }} !"
+
+  terminate-arm64:
```

Woops missed that one, this should always run even if the tests fail
```suggestion
  terminate-arm64:
    if: always()
```

---

I added it with brackets in [5195485](https://github.com/solarwindscloud/solarwinds-apm-python/pull/154/commits/51954854fd5e26a6eba311f50a4dc8bf9061f6e5) like the [repo readme](https://github.com/solarwindscloud/ec2-runner-action).